manager/src/main/resources/application.properties

```diff
@@ -45,3 +45,23 @@ quarkus.native.resources.includes=exception/exceptionInfo.json
 %dev.quarkus.iam.aws.credentials.type=static
 %dev.quarkus.iam.aws.credentials.static-provider.access-key-id=testing
 %dev.quarkus.iam.aws.credentials.static-provider.secret-access-key=testing
+
+
+# The Kafka broker location (defaults to localhost:9092)
+# This shouldn't be put in the properties see https://issues.redhat.com/browse/MGDOBR-197
+kafka.bootstrap.servers=${KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
+kafka.client.id=${KAFKA_CLIENT_ID:fake_id}
+kafka.client.secret=${KAFKA_CLIENT_SECRET:fake_secret}
+kafka.sasl.mechanism=PLAIN
+kafka.security.protocol=SASL_SSL
+kafka.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="${KAFKA_CLIENT_ID:fake_id}" password="${KAFKA_CLIENT_SECRET:fake_secret}";
+
+# Managed Connectors
+managed-connectors.cluster.id=${MC_CLUSTER_ID:managed-connectors.cluster.id}
```

Comment:
@lucamolteni Can you please add the values required for our demo environment to the `Kustomize` overlays, otherwise as soon as we deploy this to our demo cluster, the environment will break.

Let me or @r00ta know if you need help doing that.